fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/Statistics.java

```diff
@@ -95,6 +96,25 @@ public ColumnRefSet getUsedColumns() {
         return usedColumns;
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
why needs this ?

Reply:
used to judge whether Statistics has changed